Converters/ArcGIS/Speckle.Converters.ArcGIS3/ToHost/Raw/GeometryToHostConverter.cs

```diff
@@ -8,17 +8,14 @@ public class GeometryToHostConverter : ITypedConverter<IReadOnlyList<Base>, ACG.
 {
   private readonly ITypedConverter<List<SOG.Polyline>, ACG.Polyline> _polylineConverter;
   private readonly ITypedConverter<List<SOG.Point>, ACG.Multipoint> _multipointConverter;
-  private readonly ITypedConverter<List<SOG.Polygon>, ACG.Polygon> _polygonConverter;
```

Comment:
out of curiosity what is the difference Polygon and Polyline apart from `isClosed` prop